doc/site/modules/core/fiber.markdown

```diff
@@ -142,8 +144,9 @@ fiber.call("resume")
 
 ### **error**
 
-The error message that was passed when aborting the fiber, or `null` if the
-fiber has not been aborted.
+The error value that was passed when aborting the fiber, or `null` if the
+fiber has not been aborted.  Wren internal error values will always be
```

Comment:
We're not using double [sentence spacing](https://en.wikipedia.org/wiki/Sentence_spacing) elsewhere in the docs :wink: 

``` suggestion
fiber has not been aborted. Wren internal error values will always be
```